### DISCUSSION TITLE
Homebrew environment variables ?

### DISCUSSION THREAD

**Discussion by yesyves** [Homebrew/discussions#458] [Getting started]
Hi

My general question : I find references to some Homebrew environment variables here and there like HOMEBREW_TEMP, but is there a place where these variables are documented ?

My specific question : How do I force Homebrew to use a specific Git that is neither CLT nor Homebrew's ? CLT's git is too old on 10.11 and brew can't build it either. I have another git in my PATH but Homebrew does not seem to see it.

Thanks !

**Comment by carlocab** [comment]
Environment variables are documented in `man brew`. There is a section called `ENVIRONMENT`. Can't say anything about your question re `git`, unfortunately.

**Comment by yesyves** [comment]
Indeed ! My home-brew installation was incomplete so I finally found the manpage in Homebrew/manpages.

The workaround I found was to symlink the right git in `/Library/Developer/CommandLineTools/usr/bin` but for some reason this ceases to work at `brew update --force` during the installation process, even if `brew doctor` thinks git is fine.

**Comment by yesyves** [comment]
The problem probably comes from the way the git shim works
